[ML] Return assigned node in start/open job/datafeed response (#55473)
elastic/elasticsearch: docs/reference/ml/anomaly-detection/apis/open-job.asciidoc

  (Optional, time) Controls the time to wait until a job has opened. The default
  value is 30 minutes.

[[ml-open-job-response-body]]
==== {api-response-body-title}

`node`::

>   (string) The ID of the node that the job was opened on, or an empty string

Reviewer: I recommend splitting this into two shorter sentences and putting the conditional info at the start of the second sentence:
```suggestion
  (string) The ID of the node that the job was opened on. If the job is allowed to 
```

>   if the job is allowed to open lazily and has not yet been assigned to a node.

Reviewer: ...continuation of suggestion:
```suggestion
open lazily and has not yet been assigned to a node, this value is an empty string.
```

`opened`::

>   (boolean) For a success response this is always `true`. On failure an

Reviewer: ```suggestion
  (boolean) For a successful response, this value is always `true`. On failure, an
```

>   exception will be returned instead.

Reviewer: ```suggestion
  exception is returned instead.
```
